vdev/src/commands/meta/install_git_hooks.rs

```diff
@@ -0,0 +1,67 @@
+use anyhow::{Result, Ok};
+use std::fs::File;
+use std::io::Write;
+use std::os::unix::fs::PermissionsExt;
+use std::path::Path;
+use std::process::Command;
+use crate::git;
+use crate::app::CommandExt as _;
```

Comment:
```suggestion
use std::fs::File;
use std::io::Write;
use std::os::unix::fs::PermissionsExt;
use std::path::Path;
use std::process::Command;

use anyhow::{Result, Ok};

use crate::app::CommandExt as _;
use crate::git;
```

Re: our usually grouping here, you could run the formatter with nightly options - but it's really not a big deal. I wouldn't stress too much until it's actually enforced by `rustfmt`.

Reply:
I'll forgo running this, I tried `rustfmt <file name>` and `rustfmt <file_name> --nightly` but wasn't able to obtain the same changes suggested here. Given that we mentioned this is not a big deal I won't resolve this but will keep it in mind for future PRs.